package.json

```diff
@@ -41,26 +40,24 @@
   "dependencies": {
     "@nextcloud/router": "^2.2.1",
     "@nextcloud/typings": "^1.8.0",
+    "@types/dompurify": "^3.0.5",
+    "@types/escape-html": "^1.0.4",
```

Comment:
Should be moved to dev deps 🧐

Reply:
In this case yes, as the dependencies are not used in the typings. :white_check_mark: